docs/api/openapi.yaml

```diff
@@ -0,0 +1,963 @@
+                      "description": "A reading of After the EMP by Harley Tate",
+                      "genre": "Sci-fi",
+                      "id": 2,
+                      "instance_id": 2,
+                      "record": 0,
+                      "url": "https://example.com",
+                      "image_path": "",
+                      "starts": "2019-10-21 18:31:00",
+                      "ends": "2019-10-22 10:45:00"
+                    }
+                  ]
+                },
+                "sources": {
+                  "livedj": "off",
+                  "masterdj": "off",
+                  "scheduledplay": "on"
+                }
+              }
+  /week-info:
+    get:
+      summary: Retrieve the schedule for the week
+      parameters:
+        - name: api_key
+          in: path
+          description: The API key to use for authentication
+          required: false
+      responses:
+        '200':
+          description: 200 response for default request
+          content:
+            application/json:
+              example: {
+                "monday": [
+                  {
+                    "start_timestamp": "2019-10-21 17:20:00",
+                    "end_timestamp": "2019-10-21 18:31:00",
+                    "name": "Show 1",
+                    "description": "A show",
+                    "id": 1,
+                    "instance_id": 1,
+                    "instance_description": "",
+                    "record": 0,
+                    "url": "https://example.com",
+                    "image_path": "",
+                    "starts": "2019-10-21 17:20:00",
+                    "ends": "2019-10-21 18:31:00"
+                  },
+                  {
+                    "start_timestamp": "2019-10-21 18:31:00",
+                    "end_timestamp": "2019-10-22 10:45:00",
+                    "name": "Reading",
+                    "description": "A reading of After the EMP by Harley Tate",
+                    "id": 2,
+                    "instance_id": 2,
+                    "instance_description": "",
+                    "record": 0,
+                    "url": "https://example.com",
+                    "image_path": "",
+                    "starts": "2019-10-21 18:31:00",
+                    "ends": "2019-10-22 10:45:00"
+                  }
+                ],
+                "tuesday": [],
+                "wednesday": [],
+                "thursday": [],
+                "friday": [],
+                "saturday": [],
+                "sunday": [],
+                "nextmonday": [],
+                "nexttuesday": [],
+                "nextwednesday": [],
+                "nextthursday": [],
+                "nextfriday": [],
+                "nextsaturday": [],
+                "nextsunday": [],
+                "AIRTIME_API_VERSION": "1.1"
+              }
+  /station-metadata:
+    get:
+      summary: BROKEN: Retrieve the schedule for the week
```

Comment:
This string need to be quoted, it breaks when trying to parse it with swagger-ui or redoc.
Using `:` in the string can lead to weird behavior.

There might be some other strings that need to be fixed.

Reply:
Thanks. Fixed